akka-http-caching/src/main/resources/reference.conf

```diff
@@ -0,0 +1,15 @@
+akka.http.caching {
+
+  # default configuration values for LfuCache
+  lfu-cache {
+
```

Comment:
```
# Maximum number of entries the cache may store.
# After the maximum capacity is reached new entries cause old ones
# to be evicted in a last-frequently-used manner, i.e. the entries that
# are accessed most frequently are kept in the cache.

Reply:
Note that Caffeine is not an LFU, but rather frequency-biased (rather than recency-biased). In the JavaDoc we try to avoid being explicit as we want the flexibility to make changes later.

> the cache evicts entries that are less likely to be used again. For example, the cache may evict an entry because it hasn't been used recently or very often.

Reply:
Thanks for the clarification. Using the LFU abbreviation in the name is misleading.